packages/react-magma-dom/src/components/TreeView/TreeView.stories.tsx

```diff
@@ -1174,6 +1178,21 @@ const AccordionSectionWithTreeView = (props: any) => {
     }
   };
 
+  const toggleExpandAll = () => {
+    if (!isShowAll) {
+      apiRef.current?.showMore();
+      setIsShowAll(true);
+    }
+
+    setTimeout(() => {
+      apiRef.current?.expandAll();
+    }, 50);
```

Comment:
I think it needs a slightly longer timeout?? what do you think?

Reply:
On second thought, does it need the timeout at all? I don't see a difference on my machine honestly.